wiki/Bancho_(server)/it.md

```diff
@@ -0,0 +1,26 @@
+---
+tags:
+  - infrastructure
+---
+
+# Bancho (server)
+
+*Per altri usi, vedi [Bancho (disambiguazione)](/wiki/Disambiguation/Bancho).*
+
+**Bancho** (a volte *osu!Bancho*) è il nome in codice del componente del server che gestisce le interazioni in tempo reale tra i giocatori che utilizzano versioni stabili di osu!:
+
+- Sistema di Chat, [presenza online](/wiki/Client/Interface/Chat_console#extended-chat-console), e [IRC gateway](/wiki/Community/Internet_Relay_Chat)
```

Comment:
```suggestion
- Sistema di Chat, [presenza online](/wiki/Client/Interface/Chat_console#chat-console-estesa), e [IRC gateway](/wiki/Community/Internet_Relay_Chat)
```

Reply:
This has been fixed in #9032, please add `SKIP_WIKILINK_CHECK` in the PR description